lib/src/list.dart

```diff
@@ -326,3 +332,18 @@ class ListNotificationsController<T extends Object?> extends NotificationsContro
     streamController.addError(error);
   }
 }
+
+extension ListExtension<T> on List<T> {
+  /// Move the element at [from] to [to].
```

Comment:
```suggestion
  /// Move the element at [from] index to [to] index.
```

Reply:
I could also rename the arguments to `fromIndex` and `toIndex`. Would that read better?